docs/rfcs/0000-rfc-template.md

```diff
@@ -0,0 +1,59 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+# [RFC] Title here
+
+> Delete lines starting with ">". Remove sections that are not applicable.
+
+## Overview
+
+> If someone only reads this, what do you want them to know?
+
+## Goals and Non-Goals
+
+> What problems are you trying to solve?
```

Comment:
This question is pretty similar to `> Why is this change being proposed?` (in Background Motivation)

Reply:
Done